huaweicloud/resource_huaweicloud_nat_snat_rule_v2.go

```diff
@@ -67,6 +70,10 @@ func ResourceNatSnatRuleV2() *schema.Resource {
 				ForceNew:     true,
 				ExactlyOneOf: []string{"subnet_id", "network_id"},
 			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
should we add a validatefunc here as there are some limitations in the doc

Reply:
the limitation is too simple, so we can ignore the validatation in code.